phpdotnet/phd/Format/Abstract/XHTML.php

```diff
@@ -2,7 +2,7 @@
 namespace phpdotnet\phd;
 
 abstract class Format_Abstract_XHTML extends Format {
-    public $role = false;
+    public $role = [];
```

Comment:
Add a comment that this is a FILO stack, maybe also make it private and add methods to push/pop from it where all accesses to $role must go through?